migration/sql-files/0002-alter-env-add-notnull.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE environments ALTER COLUMN name SET NOT NULL;
```

Comment:
Theoretically we can already have data in DB with NULL values. So, migration could fail. It would be safer to set the existing values to something (like "stage") before setting it to NOT NULL.

Reply:
This make sense, so we are making three column from NULL to NOT NULL.  What would be the default for other two ??
- type = stage
- name = ??
- cluster-url = ??

Reply:
any data really. Just pick some cluster URL, let's say east-2.

Reply:
Done [06f8bfb](https://github.com/fabric8-services/fabric8-env/pull/26/commits/87f69ab6a7200592cf233d463567d097706f8bfb)